src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -26,7 +27,13 @@ const TopicDomainHeader = (props) => {
 
   return (
     <View>
-      <Text style={styles.domainText(shouldDisplay)} numberOfLines={1} ellipsizeMode="tail">
+      <Text
+        style={[
+          styles.domainText(shouldDisplay),
+          isPreview ? {fontSize: normalizeFontSize(12)} : {}
+        ]}
+        numberOfLines={1}
+        ellipsizeMode="tail">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional styling based on the `isPreview` prop is a good approach to handle different scenarios. However, it would be better to move this logic into the `styles.domainText` function to keep the JSX clean and readable.
```diff
- style={[
-   styles.domainText(shouldDisplay),
-   isPreview ? {fontSize: normalizeFontSize(12)} : {}
- ]}
+ style={styles.domainText(shouldDisplay, isPreview)}
```
Then in your styles definition:
```diff
domainText: (shouldDisplay, isPreview) => ({
  fontSize: isPreview ? normalizeFontSize(12) : normalizeFontSize(14),
  // other styles...
}),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->